api/app/fba_calculator.py

```diff
@@ -70,20 +79,23 @@ class FBACalculatorWeatherStation():  # pylint: disable=too-many-instance-attrib
     """ Inputs for Fire Behaviour Advisory Calculator """
 
     def __init__(self,  # pylint: disable=too-many-arguments, too-many-locals
-                 elevation: int, fuel_type: str,
+                 elevation: int, fuel_type: FuelTypeEnum,
                  time_of_interest: date, percentage_conifer: float,
                  percentage_dead_balsam_fir: float, grass_cure: float,
-                 crown_base_height: int, lat: float, long: float, bui: float, ffmc: float, isi: float,
-                 wind_speed: float, wind_direction: float, temperature: float, relative_humidity: float,
-                 precipitation: float, status: str, prev_day_daily_ffmc: float,
-                 last_observed_morning_rh_values: dict):
+                 crown_base_height: int, crown_fuel_load: Optional[float], lat: float, long: float,
+                 bui: float, ffmc: float, isi: float, wind_speed: float, wind_direction: float,
+                 temperature: float, relative_humidity: float, precipitation: float, status: str,
+                 prev_day_daily_ffmc: float, last_observed_morning_rh_values: dict):
         self.elevation = elevation
         self.fuel_type = fuel_type
         self.time_of_interest = time_of_interest
         self.percentage_conifer = percentage_conifer
         self.percentage_dead_balsam_fir = percentage_dead_balsam_fir
         self.grass_cure = grass_cure
         self.crown_base_height = crown_base_height
+        # not many people know/care about the crown fuel load, so we can fill this out with default
```

Comment:
I'm cool with this assumption if acceptable, I just don't have any idea if it is or not.

Reply:
yeah honestly I don't know either - it was a Sy addition. I'm inclined to trust his judgment on this though